docs/reference/technology/storage.md

```diff
@@ -26,7 +26,7 @@ Interfaces from low-level to high-level:
 An example of implementations of the interfaces are:
 
 - BlockDevice.
-   - QSPIBlockDevice - Block device driver for NOR-based QSPI Flash devices that support the SFDP standard. 
+   - QSPIBlockDevice - Block device driver for NOR-based QSPI Flash devices that support the SFDP standard.
```

Comment:
I don't like these dashes. Would rather have two layers of : than inconsistency.